docs/src/charm/reference/versions/1.32.md

```diff
@@ -0,0 +1,106 @@
+# 1.32
+
+**{{product}} Charms 1.32 - Release notes - 20 December 2024**
+
+Welcome to the 1.32 release of {{product}} charms, the Juju operators 
+for {{product}}! These release notes cover the highlights of this release.
+
+## What’s new
+
+<!-- add in some text on what is new in a bold -->
+- **Kubernetes 1.32** - read more about the upstream release 
+[here][upstream release].
+- **{{product}} Snap 1.32** - read more about the snap release 
+[here][snap release page].
+- **Reschedule Update Hook** - use systemd to reschedule `update-status` 
+hooks [#118].
+- **Override Installed Snap** - support a charm resource to override the 
+installed snap [#149].
+- **Snap Refresh** - allow for a snap refresh if the charm wishes to refresh 
+the same revision/channel or use a resource override [#166].
+- **Feature Configurations** - expose {{product}} snap feature config through 
+charm config [charm config].
+- **Terraform Modules** - add basic Terraform modules for the {{product}} 
+charms [#194].
+- **Upgrade Orchestration** - introduce upgrade orchestration for control 
+plane nodes [#200].
+- **Multiple Worker Integration** - allow the `k8s` charm to integrate with 
+multiple `k8s-worker` units [#221].
+
+## Fixes
+
+- Worker goes into error after control plane departed [#75][issue #75]
+- Control plane does not go into blocked when no relation to worker 
+[#90][issue #90]
+- Enable and configure `Loadbalancer` feature [#109][issue #109]
+- Enable and configure `LocalStorage` feature [#110][issue #110]
+- Support to configure Custom registries [#111][issue #111]
+- Ability to configure Cilium option `--vlan-bpf-bypass` [#112][issue #112]
+
+
+## Deprecations and API changes
+
+- Upstream
+
+For details of other deprecation notices and API changes for Kubernetes 1.32, 
+please see the
+relevant sections of the [upstream release notes][upstream-changelog-1.32].
+
+[upstream-changelog-1.32]: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.32.md#deprecation
+
+## Also in this release
+
+- Replace `AssertionError` with `ReconcileError` [#173]
+- Renames `annotations` to `cluster-annotations` in charm config [#198]
+- Add `kube-apiserver-extra-sans` option [#201]
+- Add worker `bootstrap-node-taints` setting [#215]
+- Enhance status visibility during cluster upgrades [#216]
```

Comment:
Can this join the bug fixes/ what's new section?

Reply:
I thought that this does not really fit the "bug fixes" section since these are not really bugs. Also for the "what's new?" section these are too small IMO. If you agree, I think it's best to keep them like this. It was also inspired by [what @nhennigan did for 1.32 snap](https://github.com/canonical/k8s-snap/blob/main/docs/src/snap/reference/versions/1.32.md#L28-L29).